### PR TITLE
Card updates rt

### PR DESCRIPTION
**Pull request by rtate2**

### CODE REVIEW

File: src/javascripts/helpers/data/developers.js [hunk 1]
```diff
@@ -26,27 +26,27 @@ const developers = [
...
+    email: 'mailto:cksullvn@gmail.com',
     isHired: false,
   },
   {
     picture: 'src/images/headshots/CrystalBroach.jpg',
     name: 'Crystal Broach',
     resume: 'src/resumes/CrystalBroach_Resume.pdf',
     github: 'https://github.com/broach44',
-    website: 'www.crystalbroach.com',
+    website: 'https://crystalbroach.com',
```
Comment (aclai4067) [inline src/javascripts/helpers/data/developers.js]: https isn't working, change to http://crystalbroach.com

File: src/javascripts/helpers/data/developers.js [hunk 2]
```diff
@@ -166,27 +166,27 @@ const developers = [
...
   },
   {
     picture: 'src/images/headshots/RandyTate.jpg',
     name: 'Randy Tate',
     resume: 'src/resumes/RandyTate_Resume.pdf',
     github: 'https://github.com/rtate2',
-    website: 'randytatejr.com',
-    linkedin: 'linkedin.com/in/randytatejr',
-    email: 'randytatejr@gmail.com',
+    website: 'https://randytatejr.com',
```
Comment (aclai4067) [inline src/javascripts/helpers/data/developers.js]: https technically works here, but it throws up a warning screen that the site is not private. I'd recommend changing this one to http://randytatejr.com so the user can skip that screen

**Review by aclai4067 — COMMENTED** [review]
just a couple minor url corrections and then good to merge!

**Comment by evangdesigns** [comment]
👍

**Pull request merged**